string lookup_term
---
string term_type
bool is_changeable
string term_explanation